src/pages/EmptyPage/ui/EmptyPage.tsx

```diff
@@ -0,0 +1,7 @@
+import { ContentType, EmptyBlock } from '@/widgets'
+type EmptyPageProps = {
+  content: ContentType.initial | ContentType.user
+}
+export const EmptyPage = ({ content }: EmptyPageProps) => {
```

Comment:
use FC<EmptyPageProps>
```suggestion
export const EmptyPage: FC<EmptyPageProps> = ({ content }) => {
```